hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/rollout/RolloutGrid.java

```diff
@@ -488,4 +505,16 @@ protected String createDSTooltipText(final ProxyRollout rollout) {
         }
         return tooltipText.toString();
     }
+
+    private void triggerNextRolloutGroup(final Long rolloutId, final String rolloutName,
+            final RolloutStatus rolloutStatus) {
+        if (RolloutStatus.RUNNING != rolloutStatus
+                || rolloutManagement.countGroupsByRolloutAndStatus(rolloutId, RolloutGroup.RolloutGroupStatus.SCHEDULED)
+                == 0) {
+            return;
```

Comment:
display an info like "all groups are already running" or "rollout is not in running state"